packages/fxa-settings/src/components/BrandMessaging/index.tsx

```diff
@@ -140,13 +141,7 @@ const BrandMessaging = ({
               aria-label="Close Banner"
               onClick={onClickCloseBanner}
             >
-              <Localized id="brand-close-banner" attrs={{ alt: true }}>
-                <img
-                  src="data:image/svg+xml;base64,PHN2ZyB4bWxucz0iaHR0cDovL3d3dy53My5vcmcvMjAwMC9zdmciIHdpZHRoPSIxNiIgaGVpZ2h0PSIxNiI+CiAgPGc+CiAgICA8cGF0aCBmaWxsPSJyZ2JhKDEyLCAxMiwgMTMsIC44KSIgZD0iTTkuNDE0IDhsNS4yOTMtNS4yOTNhMSAxIDAgMDAtMS40MTQtMS40MTRMOCA2LjU4NiAyLjcwNyAxLjI5M2ExIDEgMCAwMC0xLjQxNCAxLjQxNEw2LjU4NiA4bC01LjI5MyA1LjI5M2ExIDEgMCAxMDEuNDE0IDEuNDE0TDggOS40MTRsNS4yOTMgNS4yOTNhMSAxIDAgMDAxLjQxNC0xLjQxNHoiLz4KICA8L2c+Cjwvc3ZnPgo="
-                  className="cursor-pointer h-4 w-4"
-                  alt="Close Banner"
```

Comment:
So it looks like we lose the `alt=` here on the close button, which appears to be OK since it's wrapped w/ an `aria-label=` above on L140-141 anyways, although it doesn't appear like we're localizing the "Close Banner" messaging either way. 🤔